### ISSUE TITLE
Confusing $set behaviour

### ISSUE THREAD

**Issue by mokhosh** [livewire/livewire#534]
This will throw `Unable to set component data. Public property [$1] not found on component`:
```php
<a href="#" wire:click.prevent="$set('showChat',true)">
```
After a lot of head scratching I found out that this will work as expected:
```php
<a href="#" wire:click.prevent="$set('showChat', true)">
```
Not putting a space after the comma and before the `value`, will make livewire want to evaluate the string for some reason. It evaluates `true` to `1` and tries to find a public property with that name.

**To Reproduce**
Just omit the space before tha `value`.

**Expected behavior**
The space should not have any effect.

**Comment by cbaconnier** [comment]
I believe, this is related with https://github.com/livewire/livewire/issues/517

**Comment by nuernbergerA** [comment]
I will open a PR to fix this after #541 is fixed
